carma_wm/src/CARMAWorldModel.cpp

```diff
@@ -1073,6 +1090,18 @@ void CARMAWorldModel::setConfigSpeedLimit(double config_lim)
   config_speed_limit_ = config_lim;
 }
 
+void CARMAWorldModel::setVehicleParticipationType(const std::string& participant)
+{
+  if(participant == "vehicle:car")
+  {
+    participant_type_ = lanelet::Participants::VehicleCar;
+  }
+  if(participant == "vehicle:truck")
+  {
+    participant_type_ = lanelet::Participants::VehicleTruck;
+  }
```

Comment:
Again not seeing the purpose of these checks. Those are type defs of strings so this should be equivalent to 
```
participant_type_ = participant;
```